client-side/src/components/Report/htmlPDF.jsx

```diff
@@ -0,0 +1,28 @@
+import html2canvas from 'html2canvas';
+import jsPDF from 'jspdf';
+import React from 'react';
+import GenericButton from '../../stories/Button/GenericButton';
+
+const DownloadPage = ({divID}) => {
+  const downloadPDF = () => {
+    const input = document.getElementById(divID);
+    html2canvas(input, { useCORS: true }).then((canvas) => {
+      const imgData = canvas.toDataURL('image/png');
+      const pdf = new jsPDF('p', 'mm', 'a4');
+      const pdfWidth = pdf.internal.pageSize.getWidth();
+      const pdfHeight = pdf.internal.pageSize.getHeight();
+      const imgWidth = pdfWidth;
+      const imgHeight = (canvas.height * pdfWidth) / canvas.width;
+
+      pdf.addImage(imgData, 'PNG', 0, 0, imgWidth, imgHeight);
+      pdf.save('download.pdf');
+    });
+};
+  return (
+    <div>
+      <GenericButton className="secondary" label="Download as html PDF" size="medium" onClick={downloadPDF}></GenericButton>
```

Comment:
the text label should come from constants file

Reply:
I can't see that you take the label from constants.

Reply:
I changed it.